source/extensions/access_loggers/open_telemetry/grpc_access_log_impl.cc

```diff
@@ -44,6 +35,37 @@ opentelemetry::proto::common::v1::KeyValue getStringKeyValue(const std::string&
 
 } // namespace
 
+GrpcAccessLoggerImpl::GrpcAccessLoggerImpl(
+    const Grpc::RawAsyncClientSharedPtr& client,
+    const envoy::extensions::access_loggers::open_telemetry::v3::OpenTelemetryAccessLogConfig&
+        config,
+    Event::Dispatcher& dispatcher, const LocalInfo::LocalInfo& local_info, Stats::Scope& scope)
+    : GrpcAccessLogger(
+          config.common_config(), dispatcher, scope, std::nullopt,
+          std::make_unique<Common::UnaryGrpcAccessLogClient<ExportLogsServiceRequest,
+                                                            ExportLogsServiceResponse>>(
+              client,
+              *Protobuf::DescriptorPool::generated_pool()->FindMethodByName(
+                  "opentelemetry.proto.collector.logs.v1.LogsService.Export"),
+              GrpcCommon::optionalRetryPolicy(config.common_config()), genOTelCallbacksFactory())),
+      stats_({ALL_GRPC_ACCESS_LOGGER_STATS(POOL_COUNTER_PREFIX(scope, GRPC_LOG_STATS_PREFIX))}) {
+  initMessageRoot(config, local_info);
+}
+
+std::function<GrpcAccessLoggerImpl::OTelLogRequestCallbacks*()>
+GrpcAccessLoggerImpl::genOTelCallbacksFactory() {
+  return [this]() {
+    OTelLogRequestCallbacks* ptr = new OTelLogRequestCallbacks(
```

Comment:
I would use std::make_unique here to make it more idiomatic C++.

Reply:
DONE.